adapters/bidder.go

```diff
@@ -33,8 +33,9 @@ type Bidder interface {
 // TypedBid.Bid.Ext will become "response.seatbid[i].bid.ext.bidder" in the final OpenRTB response.
 // TypedBid.BidType will become "response.seatbid[i].bid.ext.prebid.type" in the final OpenRTB response.
 type TypedBid struct {
```

Comment:
Do we want a comment that "`TypedBid.BidCurrency` will become ..." for clarity and completeness?

Reply:
done